SIPS/sip-63.md

```diff
@@ -0,0 +1,52 @@
+---
+sip: 63
+title: Trading Incentives
+status: WIP
+author: Kain Warwick (@kaiynne)
+discussions-to: TBC
+
+created: 2020-05-28
+---
+
+## Simple Summary
+
+This SIP proposes a change to the Feepool contract to record the fees paid by each address to enable the distribution of trading incentives.
```

Comment:
Can you reword this to not describe how the problem will be solved (via `FeePool`) but rather that it's about incentivizing trading by distributing SNX to a pool? 

Also with the abstract - I think the details of using `FeePool` is really *how* it will be solved technically and belongs in the Technical Spec, as it's an implementation detail.

Reply:
Fixed.